README.md

```
@@ -136,26 +157,16 @@
-
-  <p align="center">
-    <a href="https://angular.io">
-      <img src="https://img.shields.io/badge/Angular-17.3.11-red.svg" alt="Angular">
-    </a>
-    <a href="https://www.typescriptlang.org/">
-      <img src="https://img.shields.io/badge/TypeScript-5.2-blue.svg" alt="TypeScript">
-    </a>
-  </p>
-
+# 🖥️ Talent Mesh Frontend
+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)